app/src/main/java/com/kms/cura/view/adapter/HealthTrackerTabAdapter.java

```diff
@@ -0,0 +1,61 @@
+package com.kms.cura.view.adapter;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import com.kms.cura.view.fragment.HealthTrackerFragment;
+import com.kms.cura.view.fragment.HealthTrackerTabFragment;
+
+import java.util.ArrayList;
+
+/**
+ * Created by toanbnguyen on 6/15/2016.
+ */
+public class HealthTrackerTabAdapter extends FragmentPagerAdapter {
+
+    private static final int FRAGMENT_NUM = 2;
+    public static String title[] = {"Current", "Past"};
+    ArrayList<String> currentHealth, pastHealth;
+
+    public HealthTrackerTabAdapter(FragmentManager fm, ArrayList<String> currentHealth, ArrayList<String> pastHealth) {
+        super(fm);
+        this.currentHealth = currentHealth;
+        this.pastHealth = pastHealth;
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        if (position == 0) { // current health
```

Comment:
Instead of using 0 and 1 and have to comment here, extract them to a constant with a self-described name so we don't need to comment.